src/components/Header/Header.tsx

```diff
@@ -1,7 +1,25 @@
-import React from 'react';
+import React from 'react'
+import { NavLink } from 'react-router-dom'
+import { slide as Menu } from 'react-burger-menu'
+import './Header.scss'
+
+import Search from '../Search/Search'
 
 const Header: React.FC = () => {
-return <div>Hello World</div>
+	return (
+		<header className="Header">
+			<Menu
+				customBurgerIcon={<img src="../search.svg" alt="search icon" />}
+				customCrossIcon={<img src="../close.svg" alt="close icon" />}>
+				<Search />
```

Comment:
@amykarnaze, we'll probably start here tomorrow with the `Search` component, which will be a controlled form.

Also we'll be able to pass a custom `onClick` to the Menu so that we can trigger the pop-out from other buttons as well as the magnifying glass

Reply:
Looking forward to the search! OnClick to the menu sounds like a good plan.